tests/mock_crust/churn.rs

```diff
@@ -187,16 +227,12 @@ fn request_during_churn_node_to_group() {
 
         poll_and_resend(&mut nodes, &mut []);
 
-        // This puts the members of the dst group to the beginning of the vec.
-        sort_nodes_by_distance_to(&mut nodes, dst.name());
-
-        let num_received = nodes.iter()
-            .take(MIN_GROUP_SIZE)
-            .filter(|node| did_receive_get_request(node, src, dst, data_id, message_id))
-            .count();
+        let (quorum, num_received) = quorum_after(&nodes, src, dst, data_id, message_id);
```

Comment:
if a `merge` happens, the quorum_before is much lower than quorum_after, and checking against it is meaningless.  (as we and only checking against `||` not `&&`)